src/richie/apps/courses/templates/courses/cms/fragment_organization_small_glimpse.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n cms_tags extra_tags thumbnail %}
```

Comment:
since this is a massive copy/paster, have you tried factorizing it and forcing a context variable like it's done for categories (tag, icon...)?

Reply:
i have not. I'll look into it.

Reply:
I still need to call a different template because it's optional and not based on context.
I now use the fragment_organization more efficiently.